Main/src/Main/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,200 @@
+package Main;
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    int water = 400;
+    int milk = 540;
+    int coffeeBeans = 120;
+    int disCups = 9;
+    int money = 550;
+    String buy = "buy";
+    String fill = "fill";
+    String take = "take";
+    String exit = "exit";
+    String remaining = "remaining";
+    String back = "back";
+    int mode = 1;
+
+
+    // method myMethod check coffee machine mode, and call appropriate method
+    public void myMethod(String inputString) {
+
+        switch (mode) {
+            case 1:
+                action(inputString);
+                break;
+            case 2:
+                buy();
+                break;
+        }
+    }
+
+    // action method which check input, change machine mode and call appropriate method
+    public void action(String actionType) {
+
+        switch (actionType) {
+            case "buy":
+                mode = 2;
+                buy();
+                break;
+            case "fill":
+                fill();
+                break;
+            case "take":
+                take();
+                break;
+            case "remaining":
+                remaining();
+                break;
+            case "exit":
+                exit();
+                break;
+        }
+    }
+
+    // method buy asks to input which coffee wants to make,or if change the mind type back
+    public void buy() {
+        mode = 1;
+        System.out.println("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino, back - to main menu:");
+        String coffeeType = CoffeeMachineSimulatorMain.getString();
+        switch (coffeeType) {
+            case "1":
+                espresso();
+                break;
+            case "3":
+                cappuccino();
+                break;
+            case "2":
+                latte();
+                break;
+            case "back":
+                System.out.println("Write action (buy, fill, take, remaining, exit):");
+                action(CoffeeMachineSimulatorMain.getString());
+                break;
+
+        }
+
+
+    }
+
+    //method espresso which checks if available ingredients, makes a coffee,
```

Comment:
@GayaneGulanyan jan you can use java doc comments write `/**` press enter , comment will be auto generated and then add your comment in that space

Reply:
thank you